File: twiml/voice/gather/gather-1/gather-1.5.x.php
<?php
require_once './vendor/autoload.php';
use Twilio\TwiML\VoiceResponse;

$response = new VoiceResponse();
$gather = $response->gather(['input' => 'speech dtmf', 'timeout' => 3,
    'numDigits' => 1]);
$gather->say('Please press 1 or say sales for sales.');

echo $response;

File: twiml/voice/gather/gather-1/gather-1.5.x.rb
require 'twilio-ruby'

response = Twilio::TwiML::VoiceResponse.new
gather = Twilio::TwiML::Gather.new(input: 'speech dtmf', timeout: 3,
  num_digits: 1)
gather.say('Please press 1 or say sales for sales.')
response.append(gather)

puts response
